app/src/main/kotlin/com/simplemobiletools/gallery/pro/activities/ViewPagerActivity.kt

```diff
@@ -716,8 +715,8 @@ class ViewPagerActivity : SimpleActivity(), ViewPager.OnPageChangeListener, View
         }
     }
 
-    @RequiresApi(Build.VERSION_CODES.O)
     private fun createShortcut() {
+        if (!isOreoPlus()) return
```

Comment:
just split it into 3 lines and add brackets, keep the code readable
if (!isOreoPlus()) {
    return
}

Reply:
Done https://github.com/SimpleMobileTools/Simple-Gallery/pull/2291/commits/dd8cd61450f5dc8c146fbc8aad021c8cc308013d